### PR TITLE
Onclusive parse new key for event status mapping [SDCP-749]

### PR DESCRIPTION
**Pull request by devketanpro**
https://github.com/superdesk/superdesk-cp/pull/233

### CODE REVIEW

File: server/planning/feed_parsers/onclusive.py [hunk 1]
```diff
@@ -66,6 +66,8 @@ def parse(self, content, provider=None):
                 GUID_FIELD: guid,
                 ITEM_TYPE: CONTENT_TYPE.EVENT,
                 "state": CONTENT_STATE.INGESTED,
+                # used for event status mapping
+                "is_provisional": event.get("isProvisional", False),
```
Comment (petrjasek) [inline server/planning/feed_parsers/onclusive.py:70]: I'm actually wondering why not implement the logic in `set_occur_status` method and only do it in the other repo?

Comment (devketanpro) [reply server/planning/feed_parsers/onclusive.py:70]: You mean, instead of doing that logic in CP, we should apply that logic here?

Comment (petrjasek) [reply server/planning/feed_parsers/onclusive.py:70]: right

Comment (devketanpro) [reply server/planning/feed_parsers/onclusive.py:70]: Actually, I assumed that it was CP-specific, so I didn't apply it here.

**Review by petrjasek — APPROVED** [review]

**Pull request merged**